openreview/conference/helpers.py

```diff
@@ -1,9 +1,10 @@
 import openreview
 import datetime
 
-def get_conference(client, request_form_id):
+def get_conference(client, request_form_id, content_override_args={}):
```

Comment:
I would prefer to add a new revision to the note that overrides the values so we don't loose track of all the changes.